sources.go

```diff
@@ -15,6 +17,27 @@ func Args() *Pipe {
 	return Echo(s.String())
 }
 
+// ListFiles contains list of files in `path` as string per line
+// names of files are relative to `path`
+func ListFiles(path string) *Pipe {
+	if strings.ContainsAny(path, "[]^*?\\{}!") {
+		fileNames, err := filepath.Glob(path)
+		if err != nil {
+			return NewPipe().WithError(err)
+		}
+		return Echo(strings.Join(fileNames, "\n"))
+	}
+	files, err := ioutil.ReadDir(path)
+	if err != nil {
+		return NewPipe().WithError(err)
+	}
+	var fileNames []string
```

Comment:
Let's use `make` to set the required capacity of this slice when we create it. That's more efficient than creating it with the default capacity and allowing `append` to repeatedly resize it.

Reply:
Fixed this as well